app/webpacker/packs/nominations.js

```diff
@@ -0,0 +1,21 @@
+import accessibleAutocomplete from "accessible-autocomplete";
+
+const locationSelect = document.querySelector(
+  "[name='nomination_request_form[local_authority_id]']"
```

Comment:
Per a discussion in slack a while back, we should be using `js-` prefixed classes for JS element selection - e.g. `document.querySelector('.js-location-select')`

Reply:
`govuk_collection_select` doesn't believe in passing custom classes to its select. So I don't think I can get it to work easily. Same for the one below,

Reply:
Check out the `html_options` argument :)

Reply:
I just spent 20 minutes of my time trying to figure out how the flying hell we are meant to pass the class in. The answer? 

Well of course, by adding `class: ["your_class"]`.

Great example of the cost of using 'more robust' selectors.

On the flip side, I learned that `html_options` is deprecated, and our school search page styles were broken, so I guess it's not all down the drain.